local-tests/run_nodes.py

```diff
@@ -9,31 +9,31 @@
 
 from chainrunner import Chain, Seq, generate_keys, check_finalized
 
-nodes = 4
-workdir = '.'
-binary = '../target/release/aleph-node'
-port = 30334
-ws_port = 9944
-rpc_port = 9933
-
-phrases = ['//Alice', '//Bob', '//Charlie', '//Dave', '//Ezekiel', '//Fanny', '//George', '//Hugo']
-keys_dict = generate_keys(binary, phrases)
+NODES = 4
```

Comment:
I guess we could use some python guidelines, or code consistency check. In other files we don't have constants written in uppercase. We should create a task for that.

Reply:
After some consideration, will abandon this and revert to the non-PEP-8-compliant version.